tests/test_server.py

```diff
@@ -1,83 +1,18 @@
-from datetime import datetime, timedelta
 import typing
 
 import orjson
-import pytest
 from fastapi.testclient import TestClient
 from pytest_mock import MockerFixture
 
-from edge_proxy.settings import AppSettings, HealthCheckSettings
 from tests.fixtures.response_data import environment_1
 
 if typing.TYPE_CHECKING:
     from edge_proxy.environments import EnvironmentService
 
 
-@pytest.mark.parametrize("endpoint", ["/proxy/health", "/health"])
-def test_health_check_returns_200_if_cache_was_updated_recently(
-    mocker: MockerFixture,
-    endpoint: str,
-    client: TestClient,
-) -> None:
-    mocked_environment_service = mocker.patch("edge_proxy.server.environment_service")
-    mocked_environment_service.last_updated_at = datetime.now()
-
-    response = client.get(endpoint)
-    assert response.status_code == 200
-
-
-def test_health_check_returns_500_if_cache_was_not_updated(
-    client: TestClient,
-) -> None:
-    response = client.get("/proxy/health")
-    assert response.status_code == 500
-    assert response.json() == {
-        "status": "error",
-        "reason": "environment document(s) not updated.",
-        "last_successful_update": None,
-    }
-
-
-def test_health_check_returns_500_if_cache_is_stale(
-    mocker: MockerFixture,
-    client: TestClient,
-) -> None:
-    last_updated_at = datetime.now() - timedelta(days=10)
-    mocked_environment_service = mocker.patch("edge_proxy.server.environment_service")
-    mocked_environment_service.last_updated_at = last_updated_at
-    response = client.get("/proxy/health")
-    assert response.status_code == 500
-    assert response.json() == {
-        "status": "error",
-        "reason": "environment document(s) stale.",
-        "last_successful_update": last_updated_at.isoformat(),
-    }
-
-
-def test_health_check_returns_200_if_cache_is_stale_and_health_check_configured_correctly(
-    mocker: MockerFixture,
-    client: TestClient,
-) -> None:
-    # Given
-    settings = AppSettings(
-        health_check=HealthCheckSettings(environment_update_grace_period_seconds=None)
-    )
-    mocker.patch("edge_proxy.server.settings", settings)
-
-    last_updated_at = datetime.now() - timedelta(days=10)
-    mocked_environment_service = mocker.patch("edge_proxy.server.environment_service")
-    mocked_environment_service.last_updated_at = last_updated_at
-
-    # When
-    response = client.get("/proxy/health")
-
-    # Then
+def test_liveness_check(client: TestClient) -> None:
+    response = client.get("/proxy/health/liveness")
```

Comment:
I'm not sure why this lives in `test_server` when the readiness checks moved to `test_health` ?

Reply:
It's because I'm using the module-global `pytestmark` variable in the `test_health` file, so I can't have this liveness test in the same file. I can rename that to `test_readiness` so it's clearer.

Reply:
If I'm being picky, I'd still rather have `test_health` and just move the parametrisation to each individual test, or write a fixture that parameterises them perhaps.

Reply:
or... if you'd rather, we can split health up into `test_health_readiness` and `test_health_liveness` where `test_health_liveness` has this one test in it. 

I think the main issue stems from the fact that `test_server` is a pretty bad module name, but I think either of the above would help the situation.

Reply:
Moved all health tests to `test_health` 👍

Reply:
Thanks - looks good 👌